utils/databake/src/converter.rs

```diff
@@ -0,0 +1,261 @@
+//! Helper structs to generate a different output than given input.
+//!
+//! This allows e.g. to store a `String` but output a `&'static str`.
+
+use crate::{Bake, BakeSize, CrateEnv};
+use core::marker::PhantomData;
+use core::ops::{Deref, DerefMut};
+use proc_macro2::TokenStream;
+use quote::quote;
+
+/// Let [`Bake`] output a `&'static str` for anything that can be `AsRef<str>`.
+///
+/// It can be helpful if the type needs to be constructed (e.g. [`String`], [`Cow<str>`](std::borrow::Cow)) but should result in a `&'static str`.
```

Comment:
nit: `Cow<str>` is already `Bake` (as `Cow::Borrowed("content")`), `Vec` would be a better second example

Reply:
the Cow implementation of bake generates a Cow, this one a &'static str. But I can simply remove it.